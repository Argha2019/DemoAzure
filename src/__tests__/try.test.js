import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ButtonGroup from '../components/pages/try.jsx';

describe('ButtonGroup', () => {
  test('it renders three buttons with the first one active', () => {
    const wrapper = shallow(<ButtonGroup />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});