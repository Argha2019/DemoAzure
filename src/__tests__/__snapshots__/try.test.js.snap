// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup it renders three buttons with the first one active 1`] = `
<div>
  <Button
    active={false}
    disabled={false}
    type="submit"
    variant="primary"
  >
    Option 1
  </Button>
  <Button
    active={false}
    disabled={false}
    type="submit"
    variant="primary"
  >
    Option 2
  </Button>
  <Button
    active={false}
    disabled={false}
    type="submit"
    variant="primary"
  >
    Option 3
  </Button>
</div>
`;
